src/lib.rs

```diff
@@ -298,7 +303,7 @@ impl Message {
     /// ```ignore
     /// let signer: Vec<u8> = message.verify_eip191(&signature)?;
     /// ```
-    pub fn verify_eip191(&self, sig: &[u8; 65]) -> Result<Vec<u8>, VerificationError> {
+    pub async fn verify_eip191(&self, sig: &[u8]) -> Result<(), VerificationError> {
```

Comment:
Should `verify_eip191` also perform eip1271 verification? Maybe we should only do eip1271 in the `verify` function?

Reply:
Yes, coming in a follow-up commit/PR with the config object